src/bin/scheduler/scheduling/plans.rs

```diff
@@ -10,19 +10,27 @@ use robotmk::section::WritePiggybackSection;
 use robotmk::termination::{ContextUnrecoverable, Terminate};
 use std::fs::create_dir_all;
 
-pub fn run_plan(plan: &Plan) -> Result<(), Terminate> {
+pub fn run_plan(plan: &Plan, quiet: bool) -> Result<(), Terminate> {
```

Comment:
Let's not couple things this way. Instead, remove writing out the plan results to JSON from `run_plan` and call this functionality separately in the place where you need it.